DigitalLearningSolutions.Web.Tests/Helpers/RegisterAdminHelperTests.cs

```diff
@@ -120,6 +122,7 @@ public void IsRegisterAdminAllowed_with_correct_data_returns_true()
             A.CallTo(() => userDataService.GetAdminsByCentreId(DefaultCentreId)).MustHaveHappenedOnceExactly();
             A.CallTo(() => centresDataService.GetCentreAutoRegisterValues(DefaultCentreId))
                 .MustHaveHappenedOnceExactly();
+            A.CallTo(() => centresDataService.GetCentreDetailsById(DefaultCentreId)).MustHaveHappenedOnceExactly();
             result.Should().BeTrue();
```

Comment:
There should be a test that `IsRegisterAdminAllowed_with_inactive_centre_returns_false`